src/modules/method/application/exceptions/method-response.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum METHOD_RESPONSE {
+  METHOD_NOT_FOUND = 'METHOD not found',
```

Comment:
```suggestion
  METHOD_NOT_FOUND = 'Method not found',
```

And so for all the responses